lib/statsig/api_client.ex

```diff
@@ -52,12 +48,12 @@ defmodule Statsig.APIClient do
           |> URI.to_string()
 
         case Req.post(url: url, json: %{events: logs}, headers: headers(key)) do
-          {:ok, %{status: code}} when code < 300 ->
+          {:ok, %{status: status}} when status in 200..299 ->
             {:ok, []}
 
           {:ok, response} ->
             Logger.error(
-              "Failed to push logs: status #{response.status}, body: #{inspect(response.body)}"
+              "Failed to push logs: status #{response.status}, body: #{inspect(String.slice(inspect(response.body), 0, 200))}"
```

Comment:
nitpick: this is a little jumbled, pull out the body
```elixir
truncated_body = response.body |> inspect() |> String.slice(0..200)
```
isn't the body a string anyways?